docs/redirectTemplates/index.html

```diff
@@ -13,4 +13,4 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-<meta http-equiv="Refresh" content="0; url=RELATIVE__PATH/LATEST__VERSION/index.html">
+<meta http-equiv="Refresh" content="0; url=../release-1.4/index.html">
```

Comment:
I think this URL is wrong. It should just be `release-1.4/index.html`. But... since it's the master branch we publish the index.html and 404.html pages from, this page shouldn't ever get used and shouldn't really matter.

Reply:
Can I remove the redirectTemplate files then?